src/modules/Elsa.Tenants/Elsa.Tenants.csproj

```diff
@@ -0,0 +1,29 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net7.0</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>enable</Nullable>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.AspNetCore.Http.Abstractions" Version="2.2.0" />
+    <PackageReference Include="Microsoft.EntityFrameworkCore" Version="7.0.13" />
```

Comment:
I noticed that the Tenants module currently has a direct dependency on EF Core. Could you clarify if this is a deliberate design choice? Additionally, I'm curious about our strategy for integrating multi-tenancy with other database providers like MongoDB, Dapper, and potentially custom ones. How do we plan to accommodate these varied environments while maintaining the module's versatility?

Reply:
Indeed, that's not a good design choice I will split that to keep the tenant project agnostic of the persistence provider